src/core/utils.js

```diff
@@ -1,17 +1,3 @@
 'use strict'
 
-exports.ifRepoExists = (repo, cb) => {
-  repo.exists((err, exists) => {
-    if (err) {
-      return cb(err)
-    }
-
-    if (exists === false) {
-      return cb(new Error('no ipfs repo found'))
-    }
-
-    cb()
-  })
-}
-
```

Comment:
lets get this merged and then I can do that PR, I don't want to put even more things into here than there are already